src/functions/retryProcessing/application/RetryProcessor.ts

```diff
@@ -0,0 +1,168 @@
+import { getConnection } from '../../../common/framework/mysql/database';
+import * as mysql from 'mysql2';
+import {
+  buildSuccessfullyProcessedQuery,
+  buildErrorsToRetryQuery,
+  buildErrorsToAbortQuery,
+  buildSupportInterventionQuery,
+  buildQueueRowsToDeleteQuery,
+  buildUpdateTestResultStatusQuery,
+  buildUpdateQueueLoadStatusQuery,
+  buildUpdateQueueLoadStatusAndRetryCountQuery,
+  buildDeleteQueueRowsQuery,
+} from '../framework/database/query-builder';
+import { getRetryConfig, retryConfig } from '../framework/retryConfig';
+import { IRetryProcessor } from './IRetryProcessor';
+
+export class RetryProcessor implements IRetryProcessor {
+
+  async processRetries(): Promise<void> {
+    await getRetryConfig();
+    await this.processSuccessful();
+    await this.processErrorsToRetry(
+      retryConfig().rsisRetryCount,
+      retryConfig().notifyRetryCount,
+      retryConfig().tarsRetryCount,
+    );
+
+    await this.processErrorsToAbort(
+      retryConfig().rsisRetryCount,
+      retryConfig().notifyRetryCount,
+      retryConfig().tarsRetryCount,
+    );
+
+    await this.processSupportInterventions();
+    await this.processOldEntryCleanup(retryConfig().retryCutOffPointDays);
+  }
+
+  async processSuccessful(): Promise<void> {
+    const connection: mysql.Connection = getConnection();
```

Comment:
We probably don't need every query to open a new connection...

Reply:
`getConnection()` and `connection.end()` are now only called in the handler.